deepchem/feat/tests/test_rdkit_grid_features.py

```diff
@@ -12,448 +12,7 @@
 np.random.seed(123)
 
 
-def random_string(length, chars=None):
-  import string
-  if chars is None:
-    chars = list(string.ascii_letters + string.ascii_letters + '()[]+-.=#@/\\')
-  return ''.join(np.random.choice(chars, length))
-
-
-class TestHelperFunctions(unittest.TestCase):
-  """
-  Test helper functions defined in rdkit_grid_featurizer module.
-  """
-
-  def setUp(self):
-    # TODO test more formats for ligand
-    current_dir = os.path.dirname(os.path.realpath(__file__))
-    self.protein_file = os.path.join(current_dir, 'data',
-                                     '3ws9_protein_fixer_rdkit.pdb')
-    self.ligand_file = os.path.join(current_dir, 'data', '3ws9_ligand.sdf')
-
-  def test_load_molecule(self):
```

Comment:
Should these tests be moved to the utils test files?

Reply:
It looks like these tests are already in the utils test files.